packages/config/config/devices/0x0234/zif5031_0.62.json

```diff
@@ -0,0 +1,102 @@
+// Logic Group ZIF5031
+// DIN Mount Multiple Actuator Controller for Floor Heating
+{
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "ZIF5031",
+	"description": "DIN Rail Interface",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0111"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.62",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Multilevel Sensor Report, Input 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Multilevel Sensor Report, Input 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Multilevel Sensor Report, Input 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Multilevel Sensor Report, Input 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "templates/logic_group_template.json#status_led"
+		},
+		"2": {
+			"$import": "templates/logic_group_template.json#led_brightness"
+		},
+		"3": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Thermistor Connected To Input 1"
+		},
+		"4": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Thermistor Connected To Input 2"
+		},
+		"5": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Thermistor Connected To Input 3"
+		},
+		"6": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Thermistor Connected To Input 4"
+		},
+		"7": {
+			"$import": "templates/logic_group_template.json#input_calibration",
+			"label": "Input 1 Calibration"
+		},
+		"8": {
+			"$import": "templates/logic_group_template.json#input_calibration",
+			"label": "Input 2 Calibration"
+		},
+		"9": {
+			"$import": "templates/logic_group_template.json#input_calibration",
+			"label": "Input 3 Calibration"
+		},
+		"10": {
+			"$import": "templates/logic_group_template.json#input_calibration",
+			"label": "Input 4 Calibration"
+		},
+		"11": {
+			"$import": "templates/logic_group_template.json#input_report_interval",
+			"label": "Input 1 Report Interval"
+		},
+		"12": {
+			"$import": "templates/logic_group_template.json#input_report_interval",
+			"label": "Input 2 Report Interval"
+		},
+		"13": {
+			"$import": "templates/logic_group_template.json#input_report_interval",
+			"label": "Input 3 Report Intervals"
```

Comment:
Please proofread for consistency. Interval vs Intervals. This should say Interval.